apps/etl-func/src/adapters/__tests__/message-event.test.ts

```diff
@@ -26,19 +27,19 @@ const sendBatchMock = vi.fn(() => Promise.resolve());
 
 const eventHubProducerClient = new mocks.EventHubProducerClient();
 
-const messageEventAdapter = new EventHubEventProducer(
+const messageEventAdapter = new EventHubEventProducer<MessageEvent>(
   eventHubProducerClient,
   messageSchema,
 );
-
+const aSimpleMessageEventArray: MessageEvent[] = [aSimpleMessageEvent];
```

Comment:
Whenever it's possible I would let typescript infer the type automatically, in this case the compiler should understand that `aSimpleMessageEventArray` is of type `MessageEvent`
```suggestion
const aSimpleMessageEventArray = [aSimpleMessageEvent];
```